Lower default RAM footprint for nRF70/nRF71 Wi-Fi drivers (#30004)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

* Added the :ref:`nRF71 Series Wi-Fi driver <nrf71_wifi_fw_if>` page documenting its firmware interface.
* Updated the :ref:`wifi_drivers` page by restructuring it into separate nRF70 Series and nRF71 Series sections.

> * Lowered the default RAM footprint of the nRF70 Series and nRF71 Series Wi-Fi drivers by reducing the default values of the following Kconfig options:

Reviewer: ```suggestion
* Updated the default values of the following Kconfig options to reduce the default RAM footprint of the Wi-Fi drivers for the nRF70 and nRF71 Series:
```
